src/gs-plugin-apk/gs-plugin-apk.c

```diff
@@ -322,20 +342,25 @@ gs_plugin_app_install (GsPlugin *plugin,
 {
   g_autoptr (GError) local_error = NULL;
   GsPluginData *priv = gs_plugin_get_data (plugin);
+  g_autofree const gchar *source = NULL;
```

Comment:
I think the `const` here is a lie since you overwrite the pointer later on (but this may very well be me misunderstanding C after writing too much Rust :D)

Reply:
Actually, there the `const` means that whatever `gchar *` is pointing to (so the content of the array) cannot be changed, but you can change the pointer itself. Anyway, the `const` wasn't really needed because of the `g_strdup` in `gs_plugin_apk_get_source`, so we anyway cannot touch the data from the library. So fixed it :)